.github/workflows/publish-ghcr.yml

```diff
@@ -0,0 +1,70 @@
+#
+name: Create and publish a Docker image
+
+on:
+  workflow_dispatch:
+  
+  release:
+    types: [published]
+
+# Defines three custom environment variables for the workflow. These are used for the Container registry domain, and a name + version for the Docker image that this workflow builds.
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
```

Comment:
The `IMAGE_NAME` is defined here and then lowered in a step, seems a bit redudant. We can define this in a single step?